File: BRaVeMobile/app/src/main/java/intl/iom/bravemobile/models/CustomLookup.java
package intl.iom.bravemobile.models;

import android.content.ContentValues;

import java.util.ArrayList;
import java.util.List;

import intl.iom.bravemobile.helpers.SelectItem;

public class CustomLookup
{
    public int id;
    public String name;
    public boolean isActive;
    public List<CustomKeyPair> values;

    public CustomLookup(int id, String name, List<CustomKeyPair> values)
    {
        this.id = id;
        this.name = name;
        this.values = values;
    }

    public List<CustomKeyPairGroup> getLkpValues()
    {
        List<CustomKeyPairGroup> lkps = new ArrayList<>();

        for(CustomKeyPair g : values)
        {
            lkps.add(new CustomKeyPairGroup(g.id, id,g.name, g.isActive));
        }

        return lkps;
    }
}

File: BRaVe-Mobile-Backend/Models/Activity_Management/CustomLookup.cs
namespace BRaVe_Mobile_Backend.Models
{
    public class CustomLookup
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public bool IsActive { get; set; } = true;
        public List<CustomKeyPair> Values { get; set; }


    }
}